frontend/src/components/JiraDashboard/components/jiraAuthorization/JiraAuth.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Button, SIZE } from "baseui/button";
+
+import useAuthorize from "./useAuthorize";
+
+require("dotenv").config();
+
+const clientId = process.env.REACT_APP_CLIENT_ID_JIRA;
+const URL = `https://auth.atlassian.com/authorize?audience=api.atlassian.com&client_id=${clientId}&scope=offline_access%20read%3Ajira-user%20read%3Ajira-work%20manage%3Ajira-project%20manage%3Ajira-configuration%20write%3Ajira-work%20manage%3Ajira-webhook%20manage%3Ajira-data-provider&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fdashboard&state=jiiaa&response_type=code&prompt=consent`;
+
+const JiraAuth = () => {
+    const { showAuthPage } = useAuthorize();
+
+    return (
+        <>
+            <ul className="instruction">
+                <li className="instruction__item">
```

Comment:
create a separate Info component which can be reused and takes a info array as prop.